common/app/common/configuration.scala

```diff
@@ -402,9 +402,14 @@ class GuardianConfiguration extends Logging {
 
   object readerRevenue {
     private lazy val readerRevenueRoot = {
-      configuration.getStringProperty("readerRevenue.s3.root") getOrElse s"${environment.stage.toUpperCase}/reader-revenue"
+      configuration.getStringProperty("readerRevenue.s3.root") getOrElse s"${environment.stage.toUpperCase}"
     }
-    lazy val contributionsBannerDeployLogKey = s"$readerRevenueRoot/contributions-banner-deploy-log"
+    val location = "/reader-revenue"
+
+    val contributionsPath = location + "/contributions-banner-deploy-log"
+    val subscriptionsPath = location + "/subscriptions-banner-deploy-log"
+    lazy val contributionsBannerDeployLogKey = s"$readerRevenueRoot$contributionsPath"
+    lazy val subscriptionsBannerDeployLogKey = s"$readerRevenueRoot$subscriptionsPath"
```

Comment:
I find this configuration a little hard to grep now - I'm worried about the default of `readerRevenueRoot` no longer has `/reader-revenue` in it for example but the config version probably does?